src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -121,6 +121,6 @@
     <value>RequiresUnreferencedCodeAnalyzer</value>
   </data>
   <data name="RequiresUnreferencedCodeAnalyzerMessage" xml:space="preserve">
-    <value>Calling '{0}' which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. {1}.</value>
+    <value>Calling '{0}' which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. {1}. {2}.</value>
```

Comment:
The last dot shouldn't be there https://github.com/mono/linker/blob/master/src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs#L1289-L1297